style.css

```diff
@@ -0,0 +1,228 @@
+.mobile-header {
+  display: flex;
+  justify-content: space-between;
+}
+
+#shape {
+  position: relative;
+  width: 74px;
+  height: 74px;
+  left: 28px;
+  top: -9px;
+}
+
+#shape1 {
+  position: absolute;
+  top: 0;
+  left: 0;
+}
+
+#logo {
+  position: absolute;
+  width: 74px;
+  height: 74px;
+  left: 34px;
+  top: 0;
+  background: url(images/Shape.svg);
+  background-blend-mode: normal, overlay, normal;
+}
+
+#mobile-logo {
+  position: absolute;
+  width: 72px;
+  height: 74px;
+  left: 0;
+  top: 0;
+  background: url(images/Shape1.svg);
+  background-blend-mode: normal, overlay, normal;
+}
+
+#mobile-menu {
+  display: flex;
+  height: 78px;
+  align-items: center;
+  width: 20px;
+  background: url(images/ic_menu.png);
+  background-repeat: no-repeat;
+}
+
+.main-header {
+  display: none;
+}
+
+.projects-block h2 {
+  font-size: 4em;
+  font-family: Playfair Display, serif;
+  margin-right: 20px;
+}
+
+#primary-text h1 {
+  font-size: 50px;
+  font-family: Playfair Display, serif;
+  padding: 0 20px;
+  color: #1a2236;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 100%;
+}
+
+.content p {
+  font-family: Poppins, serif;
+  padding: 0 20px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 21px;
+  letter-spacing: -0.01em;
+  margin: 0;
+}
+
+.content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.button {
```

Comment:
- `[OPTIONAL]` Please adhere to the `Figma` design as much as possible as I can see that your button has a border and it looks not good, kindly use the `border: none;` property in order to resolve the issue.

<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-05-01 19-54-55](https://user-images.githubusercontent.com/66476329/235466886-169299db-cbf8-495f-82fa-7d0b5edd236f.png)

</td>
<td>

![Screenshot from 2023-05-01 19-10-58](https://user-images.githubusercontent.com/66476329/235466789-06e4c528-5a77-47c6-aa36-e0024981eece.png)

</td>
</tr>
</table>